tests/libpod/container.go

```diff
@@ -0,0 +1,43 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright The KubeVirt Authors.
+ *
+ */
+
+package libpod
+
+import (
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+)
+
+const (
+	computeContainer = "compute"
+)
+
+func LookupComputeContainer(pod *v1.Pod) *v1.Container {
+	return LookupContainer(pod, computeContainer)
+}
+
+func LookupContainer(pod *v1.Pod, containerName string) *v1.Container {
+	for _, container := range pod.Spec.Containers {
+		if container.Name == containerName {
+			return &container
```

Comment:
Could there be an issue caused by returning the address of the iteration variable?

In comparison to:
```go
for i := range pod.Spec.Containers {
	if pod.Spec.Containers[i].Name == containerName {
		return &pod.Spec.Containers[i]
	}
}
```

Reply:
Yes, but it will take time until we can depend on it.